tsconfig.json

```diff
@@ -1,24 +1,24 @@
 {
-    "compilerOptions": {
-        "outDir": "build",
-        "rootDir": "lib",
-        "module": "esnext",
-        "lib": ["esnext"],
-        "target": "es5",
-        "moduleResolution": "node",
-        "newLine": "LF",
-        "strict": true,
-        "noImplicitReturns": true,
-        "noImplicitThis": true,
-        "noUnusedLocals": true,
-        "noFallthroughCasesInSwitch": true,
-        "removeComments": false,
-        "forceConsistentCasingInFileNames": true,
-        "declaration": true,
-        "noUnusedParameters": true,
-        "esModuleInterop": true,
-        "allowSyntheticDefaultImports": true,
-        "sourceMap": true,
-    },
-    "include": ["lib"],
-}
\ No newline at end of file
+  "compilerOptions": {
+    "outDir": "dist",
+    "rootDir": "lib",
+    "module": "commonjs",
```

Comment:
Any reason why this was CommonJS? We break compatibility with ES Modules if we do this.

Reply:
So it transpiles `import` to `require` calls. Otherwise normal node users cannot use the module without enabling the ESM flags or using Typescript (or if we bundled it with Rollup or similar, which would result in 2 files, one for ESM one for common).

Most ESM users most likely use Babel or similar, which interops fine, and I believe that there's flag(s) in the latest Node releases that allow ESM and commonjs interop.

Reply:
Not full complete interop - see [docs](https://nodejs.org/api/esm.html#esm_code_require_code).

Reply:
Works close enough.